R/inst/rmarkdown/templates/metadata-ids/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,113 @@
+---
+title: "Specimen and individual IDs in metadata files"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output: html_document
+params:
+  fileViewId: "syn11711684"
+  dataTypes: "clinical"
+---
+
+<!-- Store command -->
+<!-- synapser::synStore(synapser::File("./reports/ampad.html", name="AMP-AD individual and specimen ID in metadata", parentId="syn8457451")) -->
+
+```{r}
+fileViewId <- params$fileViewId
+dataTypes <- paste(params$dataTypes, collapse=",")
+```
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = FALSE,
+  message = FALSE,
+  warning = FALSE,
+  results = "asis"
+)
+
+library(tidyverse)
+library(syndccutils)
+
+get_column <- function(id, selectCol, idType="individualId") {
+  source_col <- as.name(selectCol)
+  target_col <- as.name(idType)
+  d <- data.table::fread(synapser::synGet(id)$path, select=selectCol, data.table=FALSE)
+  d <- d %>%
+    rename(rlang::UQ(target_col):=rlang::UQ(source_col)) %>%
```

Comment:
This is fine for now, but I've been told it's possible `UQ()` will be deprecated in the future so for new code going forward we should switch to `!!`.